docs/usage/general.rst

```diff
@@ -5,6 +5,13 @@ Borg consists of a number of commands. Each command accepts
 a number of arguments and options and interprets various environment variables.
 The following sections will describe each command in detail.
 
+.. container:: experimental
+
+   Experimental features are marked with red-orange stripes on the sides, like this paragraph.
```

Comment:
Where this patch applies, it already is in the text. The visual highlight just makes it more obvious and helps if an experimental feature and a stable feature are described next to each other. I'm sure we'll see that soon.